### PR TITLE
doc(github): Edit new README for language and style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
Part of https://fivetran.atlassian.net/browse/RD-231518

I've edited the new README for language and style. I've left a few questions inline, too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,27 +2,28 @@
...
+| [github__monthly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__monthly_metrics)   | Each record represents a single month, enriched with metrics about PRs and issues that were created and closed that month.                            |
+| [github__quarterly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__quarterly_metrics) | Each record represents a single quarter, enriched with metrics about PRs and issues that were created and closed that quarter.                          |
 
-# 🤔 Who is the target user of this dbt package?
-This package is intended to be leveraged by individuals who want to take advantage of analysis ready models to fuel their reporting needs. The package outputs reporting tables that may be easily linked directly to your BI tool of choice for quick time to value with your GitHub data.
+# 🤔 Why should I use this dbt package?
+This package is designed for users who want to use pre-built, analysis-ready models to fuel their reporting needs. The package creates reporting tables that you link directly to your BI tool for quick time to value with your GitHub data.
```
Comment (fivetran-chloe) [inline README.md:21]: Is there another way to say “time to value”? It’s an expression that originated in the North American tech space, and I’m not sure how global its usage is.

File: README.md [hunk 2]
```diff
@@ -2,27 +2,28 @@
...
+| [github__weekly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__weekly_metrics)    | Each record represents a single week, enriched with metrics about the PRs and issues that were created and closed that week.                             |
+| [github__monthly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__monthly_metrics)   | Each record represents a single month, enriched with metrics about PRs and issues that were created and closed that month.                            |
+| [github__quarterly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__quarterly_metrics) | Each record represents a single quarter, enriched with metrics about PRs and issues that were created and closed that quarter.                          |
 
-# 🤔 Who is the target user of this dbt package?
-This package is intended to be leveraged by individuals who want to take advantage of analysis ready models to fuel their reporting needs. The package outputs reporting tables that may be easily linked directly to your BI tool of choice for quick time to value with your GitHub data.
+# 🤔 Why should I use this dbt package?
```
Comment (fivetran-chloe) [inline README.md:20]: Are there users who shouldn’t use this package? Why not?

File: README.md [hunk 3]
```diff
@@ -2,27 +2,28 @@
 ![dbt-core](https://img.shields.io/badge/dbt_core-version_>=1.0.0_<2.0.0-orange.svg)
 
 # GitHub dbt Package ([Docs](https://dbt-github.netlify.app/))
+
 # 📣 What does this dbt package do?
-This package builds off the [dbt_github_source](https://github.com/fivetran/dbt_github_source) staging models and applies transformations that enables you to better understand your GitHub issues and pull requests.  Its main focus is to enhance these two core objects with commonly used metrics. Additionally, the metrics tables let you better understand your team's velocity over time.  These metrics are available on a daily, weekly, monthly and quarterly level. 
+This pre-built, open-source package builds on the [dbt_GitHub_source dbt package](https://github.com/fivetran/dbt_github_source). It applies transformations that enhance your GitHub issues and pull requests with common metrics so you can better understand them. Additionally, the metrics tables show your team's velocity over time. These metrics are available on a daily, weekly, monthly, and quarterly level.
```
Comment (fivetran-chloe) [inline README.md:7]: You MUST run the source package before the transform package, correct? If so, we should make that clear.

File: README.md [hunk 4]
```diff
@@ -2,27 +2,28 @@
...
+This package models the data so that it can effectively answer most of your business questions. However, it will never answer _all_ of your questions. If this package doesn't quite fit your reporting needs, you can still reference the models from this package and apply your own additional custom transformations. If you think that this package should include additional models and believe other users will benefit, please refer to the [Contributions]() section to learn how to contribute to the open-source package.
 
 # 🎯 How do I use the dbt package?
-To effectively install this package and leverage the pre-made models, you will want to follow the below steps:
+To install this package and leverage the pre-built models, do the following:
```
Comment (fivetran-chloe) [inline README.md:26]: What do you think about making a Prerequisites section, similar to the ones we have for connectors? https://fivetran.com/docs/applications/github/setup-guide#prerequisites

File: README.md [hunk 5]
```diff
@@ -2,27 +2,28 @@
...
 ## Step 1: Requirements
```
Comment (fivetran-chloe) [inline README.md:27]: What do you think about making a Prerequisites section, similar to the ones we have for connectors? https://fivetran.com/docs/applications/github/setup-guide#prerequisites

File: README.md [hunk 6]
```diff
@@ -2,27 +2,28 @@
 ![dbt-core](https://img.shields.io/badge/dbt_core-version_>=1.0.0_<2.0.0-orange.svg)
 
 # GitHub dbt Package ([Docs](https://dbt-github.netlify.app/))
```
Comment (fivetran-chloe) [inline README.md:4]: We use the term “data model” instead of “dbt package” in the docs - what do you think about using it here too? Or is “dbt package” a common enough term that we should keep it?

File: README.md [hunk 7]
```diff
@@ -88,16 +95,17 @@ models:
...
+### Step 5: Finish Setup
+You have successfully set up your dbt project to run the dbt package models! You can now execute `dbt run` and `dbt test` to see the models materialize in your destination and execute the data integrity tests applied within the package.
+
+### (Optional) Step 6: Orchestrate your package models with Fivetran
+Fivetran offers the ability for you to orchestrate your dbt project within the Fivetran dashboard. Learn how to orchestrate dbt models in your dashboard in the [Transformations for dbt Core documentation](https://fivetran.com/docs/transformations/dbt).
 
-## (Optional) Step 6: Orchestrate your package models with Fivetran
-Fivetran offers the ability for you to orchestrate your dbt project through the [Fivetran Transformations for dbt Core](https://fivetran.com/docs/transformations/dbt) product. Refer to the linked docs for more information on how to setup your project for orchestration through Fivetran. 
+## Package Dependency Matrix
```
Comment (fivetran-chloe) [inline README.md:104]: All of the Level 2 (`##`) headers before this section are questions. I think we should do this throughout for consistency and to maintain a conversational tone. What about "Does this package have dependencies?"

File: README.md [hunk 8]
```diff
@@ -107,15 +115,16 @@ packages:
   - package: dbt-labs/dbt_utils
     version: [">=0.8.0", "<0.9.0"]
 ```
-# 🙌 Contributions and Maintenance
-## Package Maintenance
-The Fivetran team maintaining this package **only** maintains the latest version of the package. We highly recommend you stay consistent with the [latest version](https://hub.getdbt.com/fivetran/github_source/latest/) of the package and refer to the [CHANGELOG](/CHANGELOG.md) and release notes for more information on changes across versions.
 
-## Contributions
+## 🙌 Contributions and Maintenance
```
Comment (fivetran-chloe) [inline README.md:119]: See my note above RE: Level 2 headers being questions. I'm stuck on a good question for this one. What about "What else should I know?"

File: README.md (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply README.md:21]: We could always just stick with "quick start". Thoughts?

File: README.md (hunk 3 above)
Comment (fivetran-joemarkiewicz) [reply README.md:7]: The source package will be run automatically if they are running the transform package. I don't think there is a need to make that clear since it will be performed for them.

File: README.md (hunk 5 above)
Comment (fivetran-joemarkiewicz) [reply README.md]: Yeah I like that! We can just swap requirements out with pre-reqs and go from there

File: README.md (hunk 6 above)
Comment (fivetran-joemarkiewicz) [reply README.md:4]: In my opinion we should keep dbt package. The people reading this will understand dbt package more and would become more confused if we start using data model instead. I think for marketing data model is good, but for our technical docs I think we should stick with package.

File: README.md (hunk 7 above)
Comment (fivetran-joemarkiewicz) [reply README.md:104]: Yeah I really like that idea! Keep the conversational tone is a great idea.

File: README.md (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply README.md]: Hmmm I don't know if we should highlight if people shouldn't. Can we instead presume that people will infer that the package isn't for them if they don't fit the "why should I use this package"?

File: README.md (hunk 1 above)
Comment (fivetran-chloe) [reply README.md:21]: @fivetran-joemarkiewicz I worry about using "quickstart" - we plan to use that word to refer to the Quickstart Data Models feature in the dashboard. What about "...directly to your BI tool so you can quickly derive value from your GitHub data"? Is that too wordy?

File: README.md (hunk 2 above)
Comment (fivetran-chloe) [reply README.md]: Yeah, that's fair. I was wondering if the package works best in specific circumstances - if there are data size limits, in certain use cases, etc. If we can be specific about that, we should.

File: README.md (hunk 3 above)
Comment (fivetran-chloe) [reply README.md:7]: Got it - you're right, let's leave it as is, then.

File: README.md (hunk 6 above)
Comment (fivetran-chloe) [reply README.md:4]: Sounds good to me!

File: README.md (hunk 7 above)
Comment (fivetran-chloe) [inline README.md:104]:
```suggestion
## Does this package have dependencies?
```

File: README.md (hunk 8 above)
Comment (fivetran-chloe) [inline README.md:119]:
```suggestion
## 🙌 What else should I know?
```

File: README.md (hunk 5 above)
Comment (fivetran-chloe) [reply README.md]: Great! I'll make the change.

**Comment by fivetran-chloe** [comment]
@fivetran-joemarkiewicz I've addressed your comments and added a Prerequisites section - please review.
